packages/protocol-parser/src/abiTypesToSchema.ts

```diff
@@ -0,0 +1,12 @@
+import { DynamicAbiType, SchemaAbiType, StaticAbiType, isDynamicAbiType } from "@latticexyz/schema-type";
+import { Schema } from "./common";
+
+export function abiTypesToSchema(abiTypes: SchemaAbiType[]): Schema {
```

Comment:
love that this is a hint towards order-preserving schemas (instead of restricted to static-then-dynamic) 👀